src/aui-layout/js/aui-layout-builder-resize-col.js

```diff
@@ -362,6 +364,26 @@ A.LayoutBuilderResizeCol.prototype = {
     },
 
     /**
+     * Protected method to set enableResizeCols atribute as false.
+     *
+     * @method _disableResizeCols
+     * @protected
+     */
+    _disableResizeCols: function() {
+        this.set('enableResizeCols', false);
+    },
+
+    /**
+     * Protected method to set enableResizeCols atribute as true.
+     *
+     * @method _enableResizeCols
+     * @protected
+     */
+    _enableResizeCols: function() {
+        this.set('enableResizeCols', true);
```

Comment:
This can cause the same problem I mentioned for removing rows, so the same fix there should be applied here.